infra/src/main/java/com/redhat/service/bridge/infra/api/APIConstants.java

```diff
@@ -20,6 +20,11 @@
      */
     public static final String SHARD_API_BASE_PATH = "/api/v1/shard/bridges/";
 
+    /**
+     * Shard realm role.
+     */
+    public static final String SHARD_REALM_ROLE = "shard";
```

Comment:
@r00ta Can I ask you to raise this at the security workstream tomorrow. IIRC the pattern of defining roles for a Shard is going to be dropped in favour of another approach of managing which Service Accounts are Shards and should have access to the API.